handlers/feedback.go

```diff
@@ -109,27 +74,17 @@ func getFeedback(w http.ResponseWriter, req *http.Request, url, errorType, descr
 		}
 	}
 
-	if len(p.Metadata.Description) > 50 {
-		p.Metadata.Description = p.Metadata.Description[len(p.Metadata.Description)-50 : len(p.Metadata.Description)]
-	}
-
-	p.ErrorType = errorType
-	p.Feedback = description
-	p.Name = name
-	p.Email = userEmail
-	p.PreviousURL = url
-
 	rend.BuildPage(w, p, "feedback")
 }
 
-// AddFeedback handles a users feedback request and sends a message to slack
-func AddFeedback(to, from string, isPositive bool, rend interfaces.Renderer, emailSender email.Sender, cacheService *cacheHelper.Helper) http.HandlerFunc {
+// AddFeedback handles a users feedback request
+func (f *Feedback) AddFeedback() http.HandlerFunc {
 	return dphandlers.ControllerHandler(func(w http.ResponseWriter, req *http.Request, lang, collectionID, accessToken string) {
-		addFeedback(w, req, isPositive, rend, emailSender, from, to, lang, cacheService)
+		addFeedback(w, req, f.Render, f.EmailSender, f.Config.FeedbackFrom, f.Config.FeedbackTo, lang, f.CacheService)
 	})
 }
 
-func addFeedback(w http.ResponseWriter, req *http.Request, isPositive bool, rend interfaces.Renderer, emailSender email.Sender, from, to, lang string, cacheService *cacheHelper.Helper) {
+func addFeedback(w http.ResponseWriter, req *http.Request, rend interfaces.Renderer, emailSender email.Sender, from, to, lang string, cacheService *cacheHelper.Helper) {
 	ctx := req.Context()
 	if err := req.ParseForm(); err != nil {
 		log.Error(ctx, "unable to parse request form", err)
```

Comment:
Appreciate this isn't part of your change, but do you think this would be better if it sent a 400 response?